### DISCUSSION TITLE
Optimize Graphs view

### DISCUSSION THREAD

**Discussion by svenvg93** [alexjustesen/speedtest-tracker#1355] [Ideas]
Hi All, 

I would like to share an idea. With the adding of the Download & Upload Latency. Two more graphs were added.
In order to minimize the amount the graphs and combine the information where useful. 
I like to propose the following layout.

- Download / Upload: the speeds combined
- Ping: High, Low, Jitter and Latency
- Upload Latency: Average, High, Low, Jitter
- Download Latency: Average, High, Low, Jitter

Still looking for a good place to show the packet loss. As well as a way to minimize the gaps if the difference between some lines are big. 

<img width="1261" alt="Scherm­afbeelding 2024-04-06 om 22 41 25" src="https://github.com/alexjustesen/speedtest-tracker/assets/4511676/0d38f625-5a99-422b-8dab-45462e73545a">
<img width="1274" alt="Scherm­afbeelding 2024-04-06 om 22 41 35" src="https://github.com/alexjustesen/speedtest-tracker/assets/4511676/19fad499-7259-4d60-9f02-4ac5084722f3">

Any other ideas are welcome of course!

**Comment by alexjustesen** [comment]
Thanks for the suggestion, a while back I separated the graphs on purpose because of the large delta some people have between upload and download speeds. Before getting fiber I had 35mbps up and 1gbps down which make the graphs essentially useless.

In the future I am planning better graphing for the embeddable dashboard but that's likely ways away. I suggest using something like Grafana if you truly want to customize your graphs.
